invenio_rdm_records/cli.py

```diff
@@ -0,0 +1,72 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2019 CERN.
+# Copyright (C) 2019 Northwestern University,
+#                    Galter Health Sciences Library & Learning Center.
+#
+# Invenio App RDM is free software; you can redistribute it and/or modify it
```

Comment:
```suggestion
# Invenio RDM Records is free software; you can redistribute it and/or modify it
```

Reply:
Thanks, used `Invenio-RDM-Records` like the others.